m/processing_tools/preprocessGliderData.m

```diff
@@ -739,6 +757,9 @@
     if isfield(ctd_choice_list, 'time')
       time_ctd_field = ctd_choice.time;
     end
+    if isfield(ctd_choice_list, 'oxygen') % -FC
+      oxygen_ctd_field = ctd_choice.oxygen;
+    end    
```

Comment:
Yes, this is exactly what I mean.
It is good practice to not pollute the code with such comments,
they can be quite cryptic for other developers and reduce readability.
You can always use the VCS capabilities to keep track of who and when did any change.